### PR TITLE
chore: update face-api

### PR DESCRIPTION
**Pull request by gabiborlea**
Update face-api with a newer version from another repository.

### CODE REVIEW

**Review by quitrk — APPROVED** [review]

**Review by saghul — COMMENTED** [review]
Do we need to update the models? I see they have been updated in the new repo.

File: webpack.config.js [hunk 1]
```diff
@@ -392,7 +392,7 @@ module.exports = (_env, argv) => {
                 ...config.plugins,
                 ...getBundleAnalyzerPlugin(analyzeBundle, 'facial-expressions-worker')
             ],
-            performance: getPerformanceHints(perfHintOptions, 1024 * 1024)
```
[the remark this answers is not in the document]
Comment (gabiborlea) [reply webpack.config.js:395]: Yes, it is aprox 1.2Mib

[the remark this answers is not in the document]
Comment (gabiborlea) [reply webpack.config.js:395]: I guess it is because some dependencies cannot be statically extracted

**Comment by gabiborlea** [comment]
> 

It works well with the old models, but I'll update them and test how it works with the ones from the new repo

**Review by saghul — APPROVED** [review]

**Pull request merged**